examples/monaco-graphql-react-vite/src/App.tsx

```diff
@@ -108,22 +108,24 @@ export default function App() {
     const variablesModel = getOrCreateModel('variables.json', defaultVariables);
     const resultsModel = getOrCreateModel('results.json', '{}');
 
-    queryEditor ??
```

Comment:
it seems the eslint rule doesn't recognize these valid expressions though? I mean I'm not worried about making the code more clear, but this code works as intended?

Reply:
it’s a valid, code works as intended, but refactoring to use `if` statement is more readable,